android/app/src/androidTest/java/org/ZingoLabs/Zingo/CommandsOnLaunch.kt

```diff
@@ -0,0 +1,98 @@
+package org.ZingoLabs.Zingo
```

Comment:
Should there be a list of asserts in this test?  Like some state that should be true after each command executes?

Reply:
yes i have started but not finished this